index.js

```diff
@@ -13,12 +13,24 @@ module.exports = function (opts) {
   mkdirp.sync(path.join(dir, 'index'))
   mkdirp.sync(path.join(dir, 'storage'))
 
-  var osm = osmdb({
-    core: kappa(dir, {valueEncoding: 'json', encryptionKey: opts.encryptionKey}),
-    index: level(path.join(dir, 'index')),
-    storage: function (name, cb) {
-      process.nextTick(cb, null, raf(path.join(dir, 'storage', name)))
+  var core = kappa(dir, {valueEncoding: 'json', encryptionKey: opts.encryptionKey})
+  var _storage = raf(path.join(dir, 'storage', name))
```

Comment:
What is `name` in this context? Here's the setup I used in multifeed:

```js
  const _storage = function (dir) {
    return function (name) {
      return raf(path.join('./whatever-base-dir', dir, name))
    }
  }
```

Reply:
right, we need a way to get that `raf` object out, will fix that.